### PR TITLE
Added blurb for https proxy

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

**Review by sundippatel — CHANGES_REQUESTED** [review]
I don't think this will be adequate by itself. What we need to do is provide instructions on what a user needs to do to get their proxy to support HTTPs requests. This involves:

1) Downloading the gateway from API Manager.
2) Opening the gateway in Studio and inserting a keystore (SSL cert).
3) Exporting and deploying the app into Runtime Manager (which we can then refer to the link specified here).
4) Enable "Last-mile security" on the application deployment.

Without a set of steps like the following, it's hard to extrapolate what's needed to achieve this outcome.

**Comment by paigehargrave** [comment]
@sundippatel Thank you! I've located the following information but not sure if it covers everything needed - please take a look and see if this will work (I'd like to avoid rewriting info whenever possible):

1. Downloading the gateway from API Manager: link to https://docs.mulesoft.com/api-manager/2.x/download-proxy-task
2. Opening the gateway in Studio and inserting a keystore (SSL cert): link to https://docs.mulesoft.com/mule-runtime/4.2/tls-configuration#use-anypoint-studio-to-configure-tls
3. Exporting and deploying the app into Runtime Manager: link to current PR link: https://docs.mulesoft.com/runtime-fabric/1.0/deploy-to-runtime-fabric
4. Enable "Last-mile security" on the application: link to: deployment.https://help.mulesoft.com/s/article/How-to-enable-last-mile-security-in-Runtime-Fabric-RTF

**Comment by paigehargrave** [comment]
@sundippatel - Added additional text - please take a look - thank you! I will run a local build to verify links, etc. if the content looks OK.

File: modules/ROOT/pages/proxy-deploy-runtime-fabric.adoc [hunk 1]
```diff
@@ -43,6 +43,14 @@ image::runtime-fabric-3.png[]
 
 After deployment, you can manage the deployed API proxy from Runtime Manager.
 
+== Enable an HTTPS Proxy
+To enable the proxy to support HTTPs requests:
+
+. Download the API gateway and deploy from API Manager using the instructions in xref:api-manager::download-proxy-task.adoc[Downloading an API Proxy].
+. Open the gateway in Anypoint Studio and insert a keystore (SSL cert) using the instructions in xref::mule-runtime::tls-configuration#use-anypoint-studio-to-configure-tls[Use Anypoint Studio to Configure TLS].
+. Export and deploy an app into Runtime Manager using the instructions in xref:deploy-to-runtime-fabric.adoc[Deploy a Mule Application to Runtime Fabric].
```
Comment (sundippatel) [inline modules/ROOT/pages/proxy-deploy-runtime-fabric.adoc]: Export and deploy the gateway using Runtime Manager...

**Review by sundippatel — DISMISSED** [review]
One line change, and then looks great!

File: modules/ROOT/pages/proxy-deploy-runtime-fabric.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/proxy-deploy-runtime-fabric.adoc]: Got it - thank you!

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**